### DISCUSSION TITLE
Installing Spacy V3 with Anaconda

### DISCUSSION THREAD

**Discussion by codingbutstillalive** [explosion/spaCy#8589] [Help: Installation] [answered]
Hello,

I used conda-forge for installation of Spacy but it installed Version 2.3.1 instead of Version 3.0. And the website does not describe how to upgrade the version using conda-forge, only via pip.

Can anyone help me here?

**Comment by polm** [comment, marked as answer]
Let me link in [your Stack Overflow question](https://stackoverflow.com/questions/68214191/how-do-i-get-spacy-version-3-0-using-anaconda). 

The [Quickstart](https://spacy.io/usage) does have a "conda" button.

![2021-07-04T13-06-19](https://user-images.githubusercontent.com/286278/124372865-ada87380-dcc8-11eb-9f13-1b1067cfafb2.png)

We had a report in #7856 that this happens if you use the base environment, but can be fixed by using any other environment. Does that fix it for you? 

If not let us know and we can look into this more.

**Reply by atulkakrana** [reply]
No this doesn't help. I am having the same issue. Following the instructions as you shared installs v2.3.5.

**Comment by Onyoursix** [comment]
I had a similar issue when installing via Conda.

After I upgraded all my packages in Conda it seemed to work, but warning - it upgrades all your packages.

`conda update --all`
